<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add_device_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add device</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_record_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="time_stamp_button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>232</y>
      <width>741</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string>Time stamp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="list_dev_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>List dev</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="device_ip">
    <property name="geometry">
     <rect>
      <x>196</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>device ip</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="device_id">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>device id</string>
    </property>
   </widget>
   <widget class="QPushButton" name="end_rec_button">
    <property name="geometry">
     <rect>
      <x>196</x>
      <y>100</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>End recording</string>
    </property>
   </widget>
   <widget class="QPushButton" name="retrieve_button">
    <property name="geometry">
     <rect>
      <x>196</x>
      <y>180</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Retrieve</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="experiment_name">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="reboot_button">
    <property name="geometry">
     <rect>
      <x>206</x>
      <y>140</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reboot</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="duration">
    <property name="geometry">
     <rect>
      <x>326</x>
      <y>140</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>12000000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="default_dev_button">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>10</y>
      <width>371</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add default devices</string>
    </property>
   </widget>
   <widget class="QPushButton" name="discard_stamp">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Discard Timestamp</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
